src/modules/site-v2/templates/_includes/phenotype-database-table.html

```diff
@@ -197,13 +192,29 @@ <h5 id="offcanvas1Label"></h5>
             sendPostRequest(data)
         })
 
+        $(document).on('click', '[id^="tagLink"]', function() {
```

Comment:
this selector is a bit too wide (`document`), try to find the closest parent to where the tag elements are.